src/phoenix/trace/fixtures.py

```diff
@@ -173,6 +178,7 @@ class TracesFixture:
 
 langchain_rag_stuff_document_chain_fixture = TracesFixture(
     name="langchain_rag_stuff_document_chain",
+    project_name="demo_langchain_rag",
```

Comment:
minor nit: might make the project names a bit more human readable since they will be displayed in the UI